docs/quickstart/install.md

```diff
@@ -36,7 +36,7 @@ Apache Zeppelin officially supports and is tested on the following environments:
   </tr>
   <tr>
     <td>OpenJDK or Oracle JDK</td>
```

Comment:
I would remove OracleJDK here. The tests are running with the Temurin OpenJDK. Perhaps it makes sense to only mention JDK 11.